Craigslist/Craigslist/Views/Listing/Detail.cshtml

```diff
@@ -1,20 +1,36 @@
-﻿@model Craigslist.Domain.Entities.Listing
-
-<div class="row">
-    <div class="col-md-6">
-        <h1>@Model.Category.Name</h1>
-        <h2 class="text-info">
-            @Model.Header</h2>
-    </div>
-    <div class="col-md-6">
-        <h4>@Model.Created</h4>
-        @Model.Price.ToString("c")</div>
-</div>
-<div class="jumbotron">
-    <blockquote>
-        <h3>@Model.Body</h3></blockquote>
-	<h3>Contact Details:</h3>
-	<p>@Model.Contact.FirstName @Model.Contact.LastName</p>
-	<p>@Model.Contact.Email</p>
-	<p class="glyphicon glyphicon-phone">@Model.Contact.Phone</p>
+﻿@model Craigslist.Domain.Entities.Listing
+<div class="container">
+    <div class="row">
+        <div class="col-md-6">

```

Comment:
Categories -> Category.

1 line: first+last names
2 line: email, phone
3 line: title + price

lines 1 to 3 - to a jumbotron

content - width 100%